src/index.css

```
@@ -15,35 +15,51 @@
   --type-scale-4: calc(var(--type-scale) * var(--type-scale-3));
   --type-scale-5: calc(var(--type-scale) * var(--type-scale-4));
 
-  --color-purple: #6200ee;
+  --variant-color-dark-pink: #e61a6b;
+  --variant-color-dark-blue: #423d89;
+  --variant-color-light-blue: #03dac6;
+  --variant-color-light-pink: #ffc4bc;
+  --variant-color-beige: #ede8d7;
+  --variant-color-black: #333333;
+  --variant-color-white: #ffffff;
+}
+
+@font-face {
+  font-family: "Graphik Web";
+  src: url("../assets/fonts/graphik/Graphik-Regular-Web.woff2") format("woff2"),
+    url("../assets/fonts/graphik/Graphik-Regular-Web.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Graphik Web Medium";
+  src: url("../assets/fonts/graphik/Graphik-Medium-Web.woff2") format("woff2"),
+    url("../assets/fonts/graphik/Graphik-Medium-Web.woff") format("woff");
 }
 
 a {
-  color: #33333d;
+  color: var(--variant-color-black);
   text-decoration: none;
-  border-bottom: 2px solid #03dac6;
+  border-bottom: 2px solid var(--variant-color-light-blue);
   transition: border-color 200ms ease-in;
 }
 a:hover {
-  border-color: #ff0166;
+  border-color: var(--variant-color-dark-pink);
 }
 
 html {
-  font-family: "IBM Plex Sans", serif;
-  /* font-size: 137.5%; */
-  font-size: calc(
-    (var(--font-size-min) * 1px) + (var(--font-size-max) - var(--font-size-min)) *
-      (
-        (100vw - (var(--viewport-min) * 1px)) /
-          (var(--viewport-max) - var(--viewport-min))
-      )
-  );
+  font-family: "Graphik Web", sans-serif;
   line-height: 1.45;
```

Burde sette default font-størrelse på html her for å fjerne forskjeller mellom oppsett. 
```suggestion
  line-height: 1.45;
  font-size: 16px;
```